README.md

```diff
@@ -1,32 +1,44 @@
-Handles End of Day (EoD) slash commands and sending EoD reminders.
-
-## Usage
-
-- `/eod`: submits an EoD
-- `/eods_left`: returns a list of who have not submitted their EoD for the day
-- `/set_eod_reminder_channel`: sets the EoD reminder channel
-- `/add_eod_reminder`: adds an EoD reminder
-- `/check_eod_reminders`: returns a list of your EoD reminders
-- `/remove_eod_reminder`: remove an EoD reminder
-
-## Initial setup
-
-Note: installation scripts only support Linux distributions at the moment
-
-1. Install systemd package if you don't have it already
-
-2. Install npm packages
-
-   `npm install`
-
-3. Run the installation script for Osteppy services
-
-   `Osteppy/src/osteppy_installation.sh`
-
-## Instructions
-
-### Set up slack token
-
-Store the Slack token as plain text in `Osteppy/config-files/SLACK_TOKEN`
-
-Note: This key is called from the backend, so it should not be available to the public.
+Server backend for Osteppy App - OSTEP that:
+
+- Manage EoDs (End of Day).
+- Check unintended open ports on research machines.
+
+## Slash commands on Slack
```

Comment:
Would it be a good idea to include `/system/load-domains` in the list since it works in production now? (Change added in #99 )

Reply:
I'm not quite sure about that endpoint. Even though we can re-load domains on the fly, it'll read the config file inside the container, not the source one. We didn't map volume for the config folder (https://github.com/Seneca-CDOT/Osteppy/blob/7499accbb6e8e7264fe58fdc1a0a339eb6550f60/Dockerfile#L44).